proto/substrait/algebra.proto

```diff
@@ -228,6 +228,20 @@ message AggregateRel {
   }
 }
 
+// The Window relational operate on a group of rows, referred to as a window, and calculate a return value for each row based on the group of rows.
+message WindowRel {
+  RelCommon common = 1;
+  Rel input = 2;
+  repeated Measure measures = 3;
+  repeated Expression partition_expressions = 4;
+  repeated SortField sorts = 5;
+  substrait.extensions.AdvancedExtension advanced_extension = 10;
+
+  message Measure {
+    Expression.WindowScalarFunction measure = 1;
+  }
```

Comment:
I don't think a dedicated message type is needed here. Also, can we call this `expressions` instead of `measures` since this is more similar to "project" than it is to "aggregate"?

In other words.  Can we change:

```
repeated Measure measures = 3;
...
message Measure {
  Expression.WindowScalarFunction measure = 1;
}
```

to...

```
repeated Expression.WindowScalarFunction expressions = 3;
```

Reply:
Updated.